Makefile

```diff
@@ -48,7 +48,7 @@ test: generate lint ## Run tests
 
 .PHONY: test-go
 test-go: ## Run tests
-	go test -v -tags=integration ./pkg/... ./cmd/...
+	go test -v -count=1 -tags=integration ./pkg/... ./cmd/...
```

Comment:
Do we still need the `-count=1` here?

Reply:
I added it so Go doesn't cache tests locally, do you prefer to have it without it?

Reply:
It shouldn't be needed under ordinary circumstances and would increase test times for iterative development. All CI jobs should not use a cache since they are inside of a container.

Reply:
Good point, I'll remove it